sunpy/net/dataretriever/sources/aia_synoptic.py

```diff
@@ -0,0 +1,64 @@
+from sunpy.net.dataretriever import GenericClient
+from sunpy.net import attrs as a
+
+__all__ = ["AIASynopticClient"]
+
+
+class AIASynopticClient(GenericClient):
+    """
+    A client for retrieving AIA synoptic data from JSOC.
+
+    This client retrieves synoptic AIA data from the following source:
+    https://jsoc1.stanford.edu/data/aia/synoptic/
+
+    The synoptic dataset includes lower-resolution (1k) images, with additional
+    image processing steps like downsampling and time integration. Characteristics
+    of the dataset can be found here: https://jsoc1.stanford.edu/data/aia/synoptic/README.html
+
+    Attributes:
+        baseurl (str): The base URL template for retrieving AIA synoptic data.
+        pattern (str): The URL pattern for AIA data, including wavelength.
+        known_wavelengths (list): A list of known wavelength codes for AIA data.
+    """
+
+    known_wavelengths = [171, 193, 211, 304, 335, 1600, 1700]
+    url_stem = r"https://jsoc1.stanford.edu/data/aia/synoptic/"
+    baseurl = url_stem + r"%Y/%m/%d/H%H00/AIA%Y%m%d_%H%M_(\d{4}).fits"
+    pattern = "{}synoptic/{year:4d}/{month:2d}/{day:2d}/H{}/AIA{}_{hour:2d}{minute:2d}_{Wavelength:4d}.fits"
+
+    required = {a.Time, a.Instrument}
+    optional = {a.Sample, a.Resolution, a.Wavelength, a.Level}
+
+    @property
+    def info_url(self):
+        return self.url_stem
+
+    @classmethod
+    def register_values(cls):
+        from sunpy.net import attrs
+
+        adict = {
+            attrs.Instrument: [
+                ("AIASynoptic", "AIA Synoptic data from the Atmospheric Imaging Assembly instrument."),
+                # ("AIA", "Data from the Atmospheric Imaging Assembly instrument."),
+            ],
+            attrs.Physobs: [
+                ("intensity", "Brightness or intensity of the solar atmosphere at different wavelengths.")
+            ],
+            attrs.Source: [("SDO", "The Solar Dynamics Observatory.")],
+            attrs.Provider: [("JSOC", "Joint Science Operations Center at Stanford.")],
+            attrs.Level: [("1.5", "Level 1.5 data processed for specialized analysis.")],
+            attrs.Wavelength: [(f"{wv:04d}", f"{wv} AA") for wv in cls.known_wavelengths],
+            attrs.Resolution: [("Synoptic", "1024x1024 dataset with 2 minute integration")],
```

Comment:
I think this should be ExtentType instead of Resolution.

Then I think we can keep Instrument as AIA.

Reply:
got it

Reply:
It seems like that gives results from both clients; the jsoc one doesn't appear to know to exclude itself when that attr is wrong. Notice also that the wavelength attr was also completely ignored from my client by not the vso client. 

Results from 2 Providers:

279 Results from the AIASynopticClient:
Source: https://jsoc1.stanford.edu/data/aia/synoptic/

       Start Time               End Time        Instrument  Physobs  Source Provider Level ExtentType Wavelength
----------------------- ----------------------- ---------- --------- ------ -------- ----- ---------- ----------
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC         94
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        131
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        171
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        193
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        211
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        304
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        335
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1600
2023-10-11 00:00:00.000 2023-10-11 00:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       4500
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC         94
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        131
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        171
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        193
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        211
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        304
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        335
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1600
2023-10-11 00:02:00.000 2023-10-11 00:02:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1700
2023-10-11 00:04:00.000 2023-10-11 00:04:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC         94
                    ...                     ...        ...       ...    ...      ...   ...        ...        ...
2023-10-11 00:56:00.000 2023-10-11 00:56:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1700
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC         94
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        131
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        171
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        193
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        211
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        304
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        335
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1600
2023-10-11 00:58:00.000 2023-10-11 00:58:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1700
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC         94
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        131
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        171
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        193
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        211
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        304
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC        335
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       1600
2023-10-11 01:00:00.000 2023-10-11 01:00:59.999        AIA intensity    SDO     JSOC   1.5   SYNOPTIC       4500
Length = 279 rows

300 Results from the VSOClient:
Source: http://vso.stanford.edu/cgi-bin/search
Total estimated size: 20.337 Gbyte

       Start Time               End Time        Source Instrument   Wavelength   Provider  Physobs  Wavetype Extent Width Extent Length Extent Type   Size  
                                                                     Angstrom                                                                        Mibyte 
----------------------- ----------------------- ------ ---------- -------------- -------- --------- -------- ------------ ------------- ----------- --------
2023-10-11 00:00:04.000 2023-10-11 00:00:05.000    SDO        AIA 193.0 .. 193.0     JSOC intensity   NARROW         4096          4096    FULLDISK 64.64844
2023-10-11 00:00:16.000 2023-10-11 00:00:17.000    SDO        AIA 193.0 .. 193.0     JSOC intensity   NARROW         4096          4096    FULLDISK 64.64844

Reply:
Is that the same dataset by from the VSO?

Reply:
No, it's one of the 4k series that triggers the staging of data when requested directly from JSOC. The problem I'm trying to solve is how to make the new dataset not appear if you don't call for it specifically, which is honestly how the Series attribute is designed to be used, and idk why this one particular product isn't done that way. I think perhaps "Level" might be the next best thing, since it is a specific pipeline/process that produces a dataset.